scripts/ci/gitlab/pipeline/publish.yml

```diff
@@ -211,7 +211,7 @@ update-node-template:
   # taking into account the 202 (as of Dec 07, 2022) publishable Substrate crates, that would equate
   # to roughly 202 minutes of delay, or 3h and 22 minutes. As such, the job needs to have a much
   # higher timeout than average.
-  timeout:                         5h
+  timeout:                         9h
```

Comment:
Is it possible to publish crates in parallel? Because if the job is going to run at night the runner might not get the latest cache for `rusty-cachier`

Reply:
> Is it possible to publish crates in parallel

Currently no. To publish a crate you need to publish all of its dependencies first, so it's a serial processing operation which happens in the context of a single job. I couldn't find a nice way to parallelize that so far.

> if the job is going to run at night the runner might not get the latest cache for rusty-cachier

Caching is not used for publishing to crates.io at the moment. It might be possible to use it in the future.